api/agent.go

```diff
@@ -824,31 +824,6 @@ func (a *Agent) ConnectCALeaf(serviceID string, q *QueryOptions) (*LeafCert, *Qu
 	return &out, qm, nil
 }
 
-// ConnectProxyConfig gets the configuration for a local managed proxy instance.
```

Comment:
For anything you removed from `agent/structs/` there is likely a mirror-universe copy of it in `api` that can go too.